criu/kerndat.c

```diff
@@ -205,16 +197,33 @@ static int kerndat_get_shmemdev(void)
 			 * OK, let's go the slower route.
 			 */
 
-			if (parse_self_maps((unsigned long)map, &dev) < 0) {
+			if (parse_self_maps((unsigned long)map, dev) < 0) {
 				pr_err("Can't read self maps\n");
-				goto err;
+				return -1;
 			}
 		} else {
 			pr_perror("Can't stat self map_files %d", e);
-			goto err;
+			return -1;
 		}
 	} else
-		dev = buf.st_dev;
+		*dev = buf.st_dev;
```

Comment:
@minhbq-99 could you please add braces for the else branch? In CRIU we try to follow the Kernel coding style:
https://www.kernel.org/doc/html/v4.10/process/coding-style.html#placing-braces-and-spaces